src/my_project/tagging/tagging_controller.py

```
@@ -78,188 +80,124 @@ def cover_getter(self):
-            logger.info("There is ONE MATCH!!!")
-            return MatchCode.ONE_MATCH
-        elif len(self.results) == 0:
-            logger.warning("There are no matches")
+        elif len(self.candidates) == 0:
             return MatchCode.NO_MATCH
-        elif len(self.results) > 1:
-            # ! Need to implement a greater detail image matcher, perhaps ORB or SIFT
-            logger.warning(f"There are multiple matches ({len(self.results)})")
-            logger.info(f"The matches are: {", ".join([g.name for g in self.results])}")  # type: ignore
-            return MatchCode.MULTIPLE_MATCHES
+            pass
```

Ruff (0.16.1)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)